src/pad.ts

```diff
@@ -9,6 +9,8 @@ import ns from './ns'
 import { Namespace, NamedNode, st } from 'rdflib'
 import { newThing, errorMessageBlock } from './widgets'
 import { beep } from './utils'
+export { notepadToHTML } from './pad-utils'
```

Comment:
I think this method should be moved into this file. (I don't think any function that are exposed to the public API should be in `-utils`-files.)